src/subgraph.ts

```diff
@@ -0,0 +1,64 @@
+import axios from 'axios';
+let SUBGRAPH_URL = '';
+
+async function getCurrentExchangeRate() {
+    const query = `
+    {
+        stakePoolExchangeRates{
+          stkBnbToBnb
+          lastUpdatedBlock
+        }
+        }
+`;
+    let data;
+    let exchangeRate;
+    await axios.post(SUBGRAPH_URL, { query }).then(result => {
+        data = result.data.data;
+        exchangeRate = data.stakePoolExchangeRates[0].stkBnbToBnb;
+    });
+    return exchangeRate;
+}
+
+async function getBlock(n: number) {
+    const query = `
+    {
+        stakePoolEpochUpdateEvents(skip:${
+            n - 1
+        },first:1  , orderBy: blockNum, orderDirection: desc) {
+          blockNum
+        }
+    }
```

Comment:
Always correctly indent the queries
```suggestion
    {
        stakePoolEpochUpdateEvents(
            skip: ${n - 1},
            first:1,
            orderBy: blockNum,
            orderDirection: desc
        ) {
            blockNum
        }
    }
```

Reply:
Done